tfs/src/main/java/hudson/plugins/tfs/TeamEventsEndpoint.java

```diff
@@ -158,8 +160,8 @@ static JSONObject innerDispatch(final String body, final String eventName, final
         }
         final AbstractHookEvent.Factory factory = factoriesByName.get(eventName);
         final Event serviceHookEvent = deserializeEvent(body);
-        final String message = serviceHookEvent.getMessage().getText();
-        final String detailedMessage = serviceHookEvent.getDetailedMessage().getText();
+        final String message = serviceHookEvent.getMessage() != null ? serviceHookEvent.getMessage().getText() : "";
+        final String detailedMessage = serviceHookEvent.getDetailedMessage() != null ? serviceHookEvent.getDetailedMessage().getText() : "";

```

Comment:
I don't see them being used either, but it would be bigger change to remove them. I only changed the code because I ran into an error here. I don't know why they were originally added.